client/src/pages/FiltersPage/FiltersPage.js

```diff
@@ -252,8 +268,45 @@ export default function FiltersPage() {
     { value: 'nonprofit', label: 'Non-profit' },
   ];
 
-  function handleToggle() {
+  function handleStarToggle() {
+    setSelectedIndex(null);
     showStarredOnly ? setShowStarredOnly(false) : setShowStarredOnly(true);
+
+    if (!showStarredOnly) {
+      const newArray = coops
+        .filter(coop => starredCoops.includes(coop.id))
+        .slice();
+      console.log('newArray:', newArray);
+      setCoopShown(newArray[0]);
+    } else {
+      setCoopShown(coops[0]);
+    }
+    //
+    // helperFunction = handleChange(useless function)
+    // helperFunction(role) to make get request
+  }
+
+  function handleChange(setter) {
+    async function helper(x) {
```

Comment:
nit: rename this to `onChange(event)`

Reply:
sorry if this was a bit ambiguous but this was referring to renaming `helper(x)` to `onChange(event)`. keep `handleChange(setter)` as it is